plugins/woocommerce/src/Blocks/Domain/Bootstrap.php

```diff
@@ -153,12 +153,15 @@ function () {
 			$this->container->get( BlockPatterns::class );
 			$this->container->get( BlockTypesController::class );
 			$this->container->get( BlockTemplatesRegistry::class )->init();
-			$this->container->get( BlockTemplatesController::class )->init();
 			$this->container->get( ClassicTemplatesCompatibility::class );
 			$this->container->get( ArchiveProductTemplatesCompatibility::class )->init();
 			$this->container->get( SingleProductTemplateCompatibility::class )->init();
 			$this->container->get( Notices::class )->init();
 			$this->container->get( PTKPatternsStore::class );
+
+			if ( wc_current_theme_is_fse_theme() ) {
```

Comment:
A classic theme might support block template parts, if I'm not wrong. So we should account for that:

```suggestion
			if ( wc_current_theme_is_fse_theme() || current_theme_supports( 'block-template-parts' ) ) {
```

Reply:
Great catch, and I'm kinda surprised that all the tests have passed without it. Should we add a basic test to cover this part?

Reply:
Yup, good point! We definitely should. Do you want to open an issue and add it to the Project board?

Reply:
Sounds good. OK if we add those tests along with this PR?

The issue is here: https://github.com/woocommerce/woocommerce/issues/48468

Reply:
Sure thing!